hazelcast/src/main/java/com/hazelcast/config/RestServerEndpointConfig.java

```diff
@@ -226,6 +228,12 @@ public RestServerEndpointConfig setSSLConfig(SSLConfig sslConfig) {
         return this;
     }
 
+    @Override
+    public RestServerEndpointConfig setAltoSocketConfig(@Nonnull AltoSocketConfig altoSocketConfig) {
```

Comment:
Preconditions.checkNotNull or something.

Reply:
Checks are done in `EndpointConfig` https://github.com/hazelcast/hazelcast/pull/23319/commits/16aaf911c270da78592443f9d81c145858b9459e